vcpkg/contributing/maintainer-guide.md

```diff
@@ -12,6 +12,29 @@ It is intended to serve the role of
 [Homebrew's Maintainer Guidelines](https://docs.brew.sh/Maintainer-Guidelines), and
 [Homebrew's Formula Cookbook](https://docs.brew.sh/Formula-Cookbook).
 
+## Overall Registry Design Goals
+
+### Ports in the current baseline must be simultaneiously installable
+
+We wish to be able to show downstream customers of libraries in the default registry that the
+combination of libraries in any given baseline we publish have been tested to work together in at
+least some configurations. Allowing ports to exclude each other breaks the ability to test such
+configurations, as the number of builds necessary for such tests would grow as
+`2^number_of_such_cases`. Moreover, installing additional dependencies is always considered "safe":
+there is no way for a port or end user to assert that a dependency is *not* installed in their
+requirements.
+
+If you wish to represent such an alternative situation for customers, consider describing how

```

Comment:
Suggestion: Use "users" rather than "customers" throughout this article since "customers" often implies "paying users".